lib/lure/pr.go

```diff
@@ -136,18 +143,27 @@ func createPullRequest(auth Authentication, sourceBranch string, destBranch stri
 
 	prRequest, err := createApiRequest(auth, "POST", fmt.Sprintf("/%s/%s/pullrequests/", owner, repo), buf)
 	if err != nil {
+		log.Println("Could not create a pull request")
 		return err
 	}
 
 	prRequest.Header.Add("Content-Type", "application/json")
 
 	log.Printf("%v\n", prRequest)
 
-	resp, err := http.DefaultClient.Do(prRequest)
+	client := pester.New()
+	client.MaxRetries = 5
+	client.Backoff = pester.ExponentialBackoff
```

Comment:
+1 but what's the initial retry interval?  retrying when you get throttled is kind of dubious unless it starts pretty high....

Reply:
https://github.com/sethgrid/pester/blob/master/pester.go#L130

Looks like 1..2..4..8..16s

Not sure I can have much impact on it, would you increase the MaxRetries?

Reply:
Hard to say. Let's see how this works out I guess

Reply:
we were retrying every second before and it was working after a few secs so I guess that should work